nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `BackOffTimeOnThrottle(TimeSpan)`: 'As A Service' products will tell there clients to back off under heavy load and so will Azure Service Bus. This setting specifies how long NServiceBus will wait to try sending again in these occurances. Defaults to 10 seconds. Note, by default this backoff period will increase exponentially during retries. 
+ * `RetryAttemptsOnThrottle(int)`: Defines how many times the transport should attempt to retry a send operation when being operated before throwing an exception.
+ * `MaximumMessageSizeInKilobytes(int)`: Specifies how large messages (and batches of messages) can be. This can differ between different types of namespaces. The default value of 256 aligns with Standard namespaces.
+ * `MessageSizePaddingPercentage(int)`: As specified in the [msdn documentation](https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.brokeredmessage.size.aspx), the Size property on a BrokeredMessage object will only provide accurate values after sending a message, which turns it pretty useless to compute a maximum batch size for batching. Therefore NServiceBus implements it's own batch size computation. Part of this computation requires a guesstimate of how much overhead will be caused by serialization of the brokered message properties. This percentage represents that guesstimate and defaults to 5%.
+ * `OversizedBrokeredMessageHandler<T>(T)`: This setting allows to override the transports behavior when a single message exceeds the maximum message size. The default behavior is represented by `ThrowOnOversizedBrokeredMessages` and will throw a `MessageTooLargeException`, informing to consider using the Databus feature instead.
+
+## Transactional behavior
+
+Under certain conditions, the Azure Service Bus transport can guarantee transactional behavior by combining the send operations and completion of the receive operation in one atomic operation. These conditions include that the source and destination are in the same namespace, that messages are received using `ReceiveMode.PeekLock` and that the transport explicitly sends messages via the receive queue. The latter can be configured using:
+
+  * `SendViaReceiveQueue(bool)`: Uses the receive queue to dispatch outgoing messages when possible. Defaults to true.
+
+## Physical Addressing Logic
+
+Next to managing connectivity and the layout of messaging entities, another important task of the transport is to figure out how these entities are named and where they can actually be found in the (seemingly infinite) cloud. This is referred to as the 'Physical Addressing Logic', and has the responsibility to turn logical endpoint names into physical addresses of Azure Service Bus entities. The following configuration sections allow to redefine this aspect of the transport.
+
+ * `UseNamespaceNamesInsteadOfConnectionStrings()`: Causes the transport to pass around namespace names instead of raw connection strings in brokered message body headers.
```

Comment:
@SeanFeldman Something is off with the naming of this setting, the default seems to perform the mapping, calling this will pass the connectionstrings through as it is